dmoj/cptbox/isolate.py

```diff
@@ -185,11 +186,15 @@ def __init__(self, read_fs, write_fs=None, writable=(1, 2)):
 
     def _compile_fs_jail(self, fs):
         if fs:
-            fs_re = '|'.join(fs)
+            fs.build()
+
```

Comment:
Excessive newlines, is this another flake8 madness? Starting to think it's something sketchy with line endings.